docs/2_registration_project_access_session.md

```diff
@@ -1,5 +1,6 @@
 # 2. Registration and project access
 
+## A. Registration
 
 
 You should have received a registration link from 
```

Comment:
Question: When are we sending the invites?

Reply:
https://scilifelab.atlassian.net/browse/DDS-2042 - we have a ticket. We didn't want to do it too early due to expiry so can either do it at the end of this week or on Monday.

Reply:
But blocker for DDS-2042 is having sent the info email to all attendees (because they need to know that the invitation is coming)